docs/guides/authentication.md

```diff
@@ -8,7 +8,7 @@ Environment variables are the only supported authentication method, and should b
 
 ```bash
 export GOOGLE_APPLICATION_CREDENTIALS_DATA="$(cat service_account.json)"
-terraform apply
+TF_LOG_PROVIDER=INFO terraform apply
```

Comment:
this seems fine to me here at least though - this command uses TPI.

Reply:
🤦🏼‍♂️ Sorry, I overlooked it!